Makefile

```diff
@@ -14,16 +14,26 @@ all: $(OUTPUT).html $(OUTPUT).txt $(OUTPUT).xml
 	$(info RFC rendering has been tested with mmark version 2.1.1 and xml2rfc 2.30.0, please ensure these are installed and recent enough.)
 
 $(OUTPUT).md: specification.markdown rfc_frontmatter.markdown rfc_backmatter.markdown EBMLSchema.xsd ebml_schema_example.xml
-	cat rfc_frontmatter.markdown $< rfc_backmatter.markdown > $(OUTPUT).md
+	cat rfc_frontmatter.markdown $< rfc_backmatter.markdown | sed "s/@BUILD_DATE@/$(shell date -I)/" > $(OUTPUT).md
```

Comment:
`-I` is an invalid option for the mac version of the `date` utility. Would `date "+%Y-%m-%d"` work for you?

Reply:
FYI: `date +'%F'` (my personal favourite) and `date +"%F"` and `dare +%F` works here.

Reply:
… and on macOS and on Windows Terminal.

Reply:
> Would date "+%Y-%m-%d" work for you?
yes
> date +'%F'
might be nicer if it works everywhere